### PR TITLE
Update public swagger

### PR DESCRIPTION
**Pull request by Yiliu-microsoft**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by amarzavery** [comment]
@Yiliu-microsoft - Please resolve merge conflicts.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json [hunk 1]
```diff
@@ -824,6 +825,10 @@
         "maxConcurrentConnections": {
           "type": "object",
           "description": "The maximum concurrent connection count for the source data store. Type: integer (or Expression with resultType integer)."
+        },
+        "disableMetricsCollection": {
+          "type": "object",
+          "description": "Specifies whether to disable collecting data source metrics. Type: boolean (or Expression with resultType boolean)."
```
Comment (amarzavery) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Can you update x-ms-examples for the operation that uses this model in the request body or response body and populate the example with this new property that has been added in this PR.

**Review by amarzavery — CHANGES_REQUESTED** [review]

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json [hunk 2]
```diff
@@ -117,7 +117,8 @@
       "properties": {
         "activity": {
           "description": "Activity name.",
-          "type": "string"
+          "type": "string",
+          "minLength": 1
```
Comment (ravbhatnagar) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: this is a breaking change. Looks like this restriction is added in the existing api-version.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json [hunk 3]
```diff
@@ -824,6 +825,10 @@
         "maxConcurrentConnections": {
           "type": "object",
           "description": "The maximum concurrent connection count for the source data store. Type: integer (or Expression with resultType integer)."
+        },
+        "disableMetricsCollection": {
```
Comment (ravbhatnagar) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: writable properties are typically added in new api-version as they can result in unintentional update of goal state when done in same api-version.

**Review by ravbhatnagar — COMMENTED** [review]
a couple of comments. please ping me in case of any questions.

**Comment by yungezz** [comment]
@Yiliu-microsoft  could you pls resolve comments and fixing CI? otherwise is it ok to close the PR for now, and pls create new PR when you're ready.

**Comment by yungezz** [comment]
close the PR since no update for a while. I cannot find corresponding email address from internal site either.

**Pull request closed without merging**